api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -348,6 +348,14 @@ message HttpConnectionManager {
     repeated HttpFilter filters = 2;
   };
   repeated UpgradeConfig upgrade_configs = 23;
+
+  // If true, the order of encoder filters will be reversed to that of filters
+  // configured in the HTTP filter chain. Otherwise, it will keep the existing
+  // order.
+  // Note: this is a bug fix for envoy, which is designed to have the reversed
```

Comment:
s/envoy/Envoy/, can you reference the GH issue here as well?

Reply:
Done.